cfme/infrastructure/host.py

```diff
@@ -92,15 +92,29 @@ def __init__(self, **kwargs):
             super(Host.Credential, self).__init__(**kwargs)
             self.ipmi = kwargs.get('ipmi')
 
-    def update(self, updates, validate_credentials=False):
+    def update(self, updates, validate_credentials=False, action='edit_from_details'):
         """Updates a host in the UI. Better to use utils.update.update context manager than call
         this directly.
 
         Args:
            updates (dict): fields that are changing.
+           action (str): denotes additional functionality. Expecting edit_from_details,
+           edit_from_hosts, delete, cancel, or nav_away.
```

Comment:
Why would you expect `delete` as an action within update? It doesn't look like its actually handled in the logic here.

Reply:
You are correct. Removing